document new job which synchronize client registrations with SCC (#926)
uyuni-project/uyuni-docs: modules/administration/pages/task-schedules.adoc

Generates Cobbler files corresponding to Kickstart profiles created by the configuration wizard.

menu:mgr-forward-registration-default:[]::

> Synchronizes client registration data with SUSE Customer Center (SCC) (new, changed or deleted clients' data are forwarded).

Reviewer: ```suggestion
Synchronizes client registration data with SUSE Customer Center (SCC).
By default, new, changed, or deleted client data are forwarded.
```

> In case synchronizing is not wanted set the following value in [path]``/etc/rhn/rhn.conf``:

Reviewer: ```suggestion
To disable synchronization set in [path]``/etc/rhn/rhn.conf``:
```
